examples/with-global-stylesheet/README.md

```diff
@@ -0,0 +1,31 @@
+README
```

Comment:
Can you port this documentation to the same style as https://github.com/zeit/next.js/tree/master/examples/basic-css. All examples have kind of the same readme ❤️